debbuild

```diff
@@ -749,13 +749,18 @@ ENDIF: if (/^\s*%endif/) {
 # an rpmbuild-compatible .spec file
       if (my ($dname,$dvalue) = /^(Summary|Group|Version|Conflicts|Provides|
           BuildArch(?:itecture)?|BuildRequires|BuildConflicts|
-          Recommends|Suggests|Enhances|Obsoletes|Breaks|Replaces|
+          Recommends|Supplements|Suggests|Enhances|Obsoletes|Breaks|Replaces|
           PreReq|Requires(?:\(pre(?:un)?\))?|Pre-Depends):\s+(.+)$/ix) {
         $dname =~ tr/[A-Z]/[a-z]/;
         $dvalue =~ s/^noarch/all/i if $dname =~ s/^BuildArch(?:itecture)?/arch/i;
         if (grep { $dname eq $_ } qw(recommends suggests enhances breaks
             replaces requires conflicts provides pre-depends)) {
           push @{$pkgdata{$subname}{$dname}}, splitreqs($dvalue);
+        } elsif (grep { $dname eq $_ } qw(supplements)) {
+          push @{$pkgdata{$subname}{enhances}}, splitreqs($dvalue);
+          warn _('Warning:  \'').
+               _("Supplements:' is not natively supported by .deb packages.\n").
+               _("Downgrading relationship to Enhances:.\n");
```

Comment:
Please mention the package name whose relationship is being downgraded. Otherwise, looks good!

Reply:
This is probably something that needs to happen as a separate refactor. We are apparently doing it this way everywhere already. :(

Reply:
Ah, I didn't realize this was in multiple places already. Fair enough.